recipes-core/images/aos-image-minimal.bb

```diff
@@ -0,0 +1,14 @@
+SUMMARY = "A small image which containing aos-vis."
```

Comment:
Change it to: "An image which contains AOS components"

Reply:
Done